artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AMQPMessageSupport.java

```diff
@@ -178,6 +182,9 @@
    public static final Binary EMPTY_BINARY = new Binary(new byte[0]);
    public static final Data EMPTY_BODY = new Data(EMPTY_BINARY);
 
+   public static final String X_OPT_PREFIX = "x-opt-";
```

Comment:
@gemmellr I saw an email from you asking about Strings defined here but I can't see the comment. (was it deleted?)

I see your point ... I needed this as SimpleString at some point, and my code changes along the way in a way I don't need it any more.. thanks... ammended.

Reply:
I did delete it yes, I hit the wrong button, was starting a review and instead hit the comment button. I've just finished the review so its sort of back now.